src/components/Letter.tsx

```diff
@@ -195,6 +216,7 @@ export function Letter({ letter, isEditable, disableDrag, idx }: Props) {
         newLetterInteractionData[color].numDrags++;
         void updateLetterInteraction(id, newLetterInteractionData);
         setDragging(false);
+        setCurrentDraggedLetter(undefined)
```

Comment:
should exit focus here to handle the focus interaction

Reply:
As in manually unfocus the element?

Reply:
yeah or we dont have the interaction for focus which i guess is also prob fine, just have the blue outline?